model-optimizer/extensions/ops/GRUCell.py

```diff
@@ -45,7 +46,8 @@ def __init__(self, graph: Graph, attrs: dict):
             'out_ports_count': 1,
             'version': 'opset3',
             'wr_input_id': 2,
-            'gates_count': 3
+            'gates_count': 3,
+            'linear_before_reset': 'false'
```

Comment:
Why string? It should be `False` Python boolean value.

Reply:
Yes, fixed.